tasks/db_deploy/orca_sql.py

```diff
@@ -584,3 +584,17 @@ def drop_drdbo_role_sql() -> TextClause:
         DROP ROLE IF EXISTS drdbo_role;
     """
     )
+
+
+def add_multipart_chunksize_sql() -> TextClause:
+    """
+    SQL that adds the multipart_chunksize_mb column to orca_files.
+
+    Returns: SQL for adding multipart_chunksize_mb.
+    """
+    return text(
+        """
+        ALTER TABLE table_name
+        ADD COLUMN multipart_chunksize_mb numeric NULL;
```

Comment:
I would change the type from `numeric` to `int` since we would want a positive integer number and `int` should be large enough. See https://www.postgresql.org/docs/13/datatype-numeric.html

Reply:
You may want to use `ADD COLUMN IF NOT EXISTS` as a little bit of a safety valve. See https://www.postgresql.org/docs/13/sql-altertable.html. This allows for migrations to be run over without any issues.

Reply:
You may also want to add the column comment information like we do when creating the table for example
`COMMENT ON COLUMN recovery_job.multipart_chunksize_mb IS 'Chunksize setting for AWS S3 copy command that was configured for the collection the granule is being restored from.'`